pages/signup.css

```diff
@@ -0,0 +1,75 @@
+@import url("https://fonts.googleapis.com/css2?family=Comfortaa:wght@700&family=Quicksand:wght@300&family=Roboto:wght@100;300;400;500&display=swap");
+
+:root {
+  --primary-color: #fff;
+
+  --secondary-color: #4b5563;
+  /* --action-color: #f26430;confused */
+  --action-color: #ff715b;
+
+  --background-color: #171717;
+}
+
+html {
+  background-color: #fdf4f3;
+}
+
+.login-container input:nth-child(2),
+input:nth-child(3),
+input:nth-child(4) {
```

Comment:
Suggestion - Since these styles are layout is common for both the places, you can create a single ccs files to reduce duplicity.